apps/mark-scan/backend/src/app.diagnostics.test.ts

```diff
@@ -150,7 +147,10 @@ test('diagnostic records', async () => {
       diagnosticType: 'mark-scan-accessible-controller',
     })
   ).toBeNull();
-  jest.useFakeTimers().setSystemTime(0);
+  vi.useFakeTimers({
+    shouldAdvanceTime: true,
```

Comment:
Does this have the potential for introducing non-determinism, given the `timestamp: 0` assertion below? Or does something else break if the fake timers stay frozen for this portion of the test?

Reply:
It seems to work, but I take your point and it seems to work without it, so I removed it.